lib/resty/dns/resolver.lua

```diff
@@ -484,6 +496,43 @@ local function parse_section(answers, section, buf, start_pos, size,
 
             pos = p
 
+        elseif typ == TYPE_NAPTR then
+            if len < 7 then
+                return nil, "bad NAPTR record value length: " .. len
+            end
+
+            local hi = byte(buf, pos)
+            local lo = byte(buf, pos + 1)
+            ans.order = lshift(hi, 8) + lo
+
+            hi = byte(buf, pos + 2)
+            lo = byte(buf, pos + 3)
+            ans.preference = lshift(hi, 8) + lo
+
+            local str, p = _decode_string(buf, pos + 4)
+            if not str then return nil, pos end
```

Comment:
Style: we prefer the following form instead for readability:

```
if not str then
    return nil, pos
end
```

Please fix other similar places as well. Thanks!